src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -71,17 +72,19 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
                 CultureUtility.DisableLocalization();
             }
 
-            var app = InitializeApp(args);
-            args = args
-                .Where(e => e != "package")
-                .ToArray();
+            log.LogLevel = LogLevel.Information;
 
-            var verbosity = app.Option(XPlatUtility.VerbosityOption, Strings.Switch_Verbosity, CommandOptionType.SingleValue);
+            var app = InitializeApp(args, log);
 
-            // Options aren't parsed until we call app.Execute(), so look directly for the verbosity option ourselves
-            LogLevel logLevel;
-            TryParseVerbosity(args, verbosity, out logLevel);
```

Comment:
Need to dive into Verbosity a bit. Verbosity doesn't fit nicely with normal verbosity values. But i need to dig into this area...and consider right move.

Reply:
investigated. only command that supports verbosity ... is "dotnet nuget".
None of the subcommands support them.
Cutting that support...as it appears to makes no sense.